clarity/contracts/alex-vault.clar

```diff
@@ -115,7 +115,7 @@
   (begin     
     (try! (check-is-approved tx-sender))
     (try! (check-is-approved-token (contract-of token)))
-    (as-contract (unwrap! (contract-call? token transfer-fixed amount tx-sender recipient none) ERR-TRANSFER-FAILED))
+    (as-contract (try! (contract-call? token transfer-fixed amount tx-sender recipient none)))
```

Comment:
Can be simplified to just the `as-contract` call. Can remove the `try!` and the `(ok true)`.